egs2/sunda/asr1/local/score.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
If this is the same as https://github.com/espnet/espnet/blob/master/egs2/open_li52/asr1/local/score.sh you can use soft link for this